pt-BR/mail/chrome/messenger/messengercompose/composeMsgs.properties

```diff
@@ -263,59 +263,59 @@ sendAnyway=Send anyway
+12597=Não foi possível autenticar no servidor SMTP %S. Por favor, verifique a senha e o 'Método de autenticação' em 'Configurações de conta | Servidor de saída (SMTP)'.
 
 ## @name NS_ERROR_SMTP_AUTH_GSSAPI
 # LOCALIZATION NOTE (12598): $S is server hostname
-12598=The Kerberos/GSSAPI ticket was not accepted by the SMTP server %S. Please check that you are logged in to the Kerberos/GSSAPI realm.
+12598=O tíquete Kerberos/GSSAPI não foi aceito pelo servidor SMTP %S. Por favor, verifique se você está logado no realm do Kerberos/GSSAPI.
 
 ## @name NS_ERROR_SMTP_AUTH_MECH_NOT_SUPPORTED
 # LOCALIZATION NOTE (12599): $S is server hostname
-12599=The SMTP server %S does not support the selected authentication method. Please change the 'Authentication method' in the 'Account Settings | Outgoing Server (SMTP)'.
+12599=O servidor SMTP %S não suporta o método de autenticação selecionado. Por favor, altere 'Método de autenticação' em 'Configurações de conta | Servidor de saída (SMTP)'.
 
 ## @name NS_ERROR_SMTP_AUTH_NOT_SUPPORTED
 # LOCALIZATION NOTE (12600): $S is server hostname
-12600=Unable to authenticate to SMTP server %S. It does not support authentication (SMTP-AUTH) but you have chosen to use authentication. Please change the 'Authentication method' to 'None' in the 'Account Settings | Outgoing Server (SMTP)' or contact your email service provider for instructions.
+12600=Não foi possível autenticar no servidor SMTP %S. Ele não suporta autenticação (SMTP-AUTH), mas você selecionou o uso dela. Por favor, altere o 'Método de autenticação' para 'Nenhum' em 'Configurações de conta | Servidor de saída (SMTP)' ou contate o seu provedor de e-mails para mais instruções.
 
 ## Strings use for the save message dialog shown when the user close a message compose window
 saveDlogTitle=Salvar mensagem
 saveDlogMessage=A mensagem não foi enviada. Deseja salvá-la na pasta “Rascunhos”?
-saveDlogSaveBtn=Save
-saveDlogDontSaveBtn=Don't Save
-saveDlogCancelBtn=Cancel
+saveDlogSaveBtn=Salvar
+saveDlogDontSaveBtn=Não salvar
+saveDlogCancelBtn=Cancelar
 
 ## generics string
 defaultSubject=(assunto em branco)
 chooseFileToAttach=Anexar arquivos
 
 ## LOCALIZATION NOTE (SaveDialogMsg): %1$S is the current subject of the compose window
-windowTitle=Compose: %1$S
+windowTitle=Criar: %1$S
 
 ## String used by the dialog that ask the user to enter a subject
-sendMsgTitle=Send Message
-subjectDlogMessage=You did not specify a subject for this message. If you would like to provide one, please type it now.
+sendMsgTitle=Enviar mensagem
+subjectDlogMessage=Você não especificou um assunto para esta mensagem. Se você desejar fornecer um, digite agora, por favor.
 
 ## String used by the dialog that informs the user about the newsgroup recipient
-recipientDlogMessage=Esta conta suporta somente destinatários de email. Caso continue, os newsgroups serão ignorados.
+recipientDlogMessage=Esta conta suporta somente destinatários de e-mail. Caso continue, os newsgroups serão ignorados.
```

Comment:
CORRECTIONS NEEDED it should be recipientDlogMessage=Esta conta suporta somente destinatários de e-mail. Caso continue, os grupos de notícias serão ignorados.